io/writeToFile.go

```diff
@@ -6,7 +6,9 @@ import (
 	"os"
 )
 
-func WriteToFile(username string, data []string) {
+func WriteToFile(username string, dataToFile chan []string) error {
```

Comment:
Write to File should only care about content like string data, should not use any channel.